tests/.gitignore

```diff
@@ -3,8 +3,10 @@
 .libs/
 Makefile
 Makefile.in
+temp.*
```

Comment:
Let's just delete temp.bmp at the end of the test instead.

Reply:
I added some code to delete the file at the end of the test, but also kept this - if the tests fail, we abort so the assets may not be cleaned up

Reply:
I don't care about that since it's not the normal case, please remove it :)